createdisk.sh

```diff
@@ -412,7 +412,7 @@ create_qemu_image $libvirtDestDir
 
 copy_additional_files $1 $libvirtDestDir
 
-tar cSf - $libvirtDestDir | xz --threads=0 >$libvirtDestDir.$crcBundleSuffix
+tar cSf - --sort=name $libvirtDestDir | xz --threads=0 >$libvirtDestDir.$crcBundleSuffix
```

Comment:
a bit fragile as this breaks as soon as we add a file which comes before `crc-bundle-info.json` in alphabetical order. The alternatives would be a bit more verbose I think :-/

Reply:
Yes sadly it's not easy. A more general solution would be to use zip (so that there is an index of files) but the archive would be 700MB bigger.

Reply:
`tar cSf - $libvirtDestDir/crc-bundle-info.json --exclude $libvirtDestDir/crc-bundle-info.json $libvirtDestDir | xz --threads=0 >$libvirtDestDir.$crcBundleSuffix` seems to do the trick, but it's on the ugly side :)